config/zones.json

```diff
@@ -1868,17 +1868,17 @@
       ]
     ],
     "capacity": {
-      "biomass": 1776,
-      "coal": 2968,
-      "gas": 12156,
+      "biomass": 1578,
+      "coal": 2977,
+      "gas": 12238,
       "geothermal": 16,
-      "hydro": 24086,
-      "hydro storage": 1728,
+      "hydro": 16947,
+      "hydro storage": 4656,
```

Comment:
These are quite a large change, has something been reclassified?

Also did solar really go down by a gigawatt?

Reply:
Well, there is evidently more hydro storage, as can be seen in the screenshot.

Reply:
And less hydro. So have some plants formerly classified as hydro been reclassified as hydro storage? Electricity Map might need to re-run data archives if so

Reply:
I'm curious how we ever had hydro storage at 1728, cannot seem to find that number anywhere